pkg/server/share_manager.go

```diff
@@ -20,12 +20,12 @@ const waitBetweenChecks = time.Second * 5
 const healthCheckInterval = time.Second * 10
 const exportPath = "/export"
 const configPath = "/tmp/vfs.conf"
-const devPath = "/dev"
+const DevPath = "/dev"
 
 type ShareManager struct {
 	logger logrus.FieldLogger
 
-	volume volume.Volume
+	Volume volume.Volume
```

Comment:
Why need to export?

Reply:
Outdated implantation. Updated. Thank you,